### PR TITLE
Pragmatic and future-proof way of keyword lookups

### PR DESCRIPTION
**Pull request by dhilipsiva**
```
from keyword import kwlist
print(kwlist)
# ['False', 'None', 'True', 'and', 'as', 'assert', 'break', 'class', 'continue', 'def', 'del', 'elif', 'else', 'except', 'finally', 'for', 'from', 'global', 'if', 'import', 'in', 'is', 'lambda', 'nonlocal', 'not', 'or', 'pass', 'raise', 'return', 'try', 'while', 'with', 'yield']
```

### CODE REVIEW

File: box.py [hunk 1]
```diff
@@ -39,10 +40,6 @@
 __author__ = 'Chris Griffith'
 __version__ = '3.0.1'
 
-ILLEGAL_ATTRIBUTES = ('if', 'elif', 'else', 'for', 'from', 'as', 'import',
```
Comment (dhilipsiva) [inline box.py]: @cdgriffith `kwlist` is a superset of  `ILLEGAL_ATTRIBUTES` with these extra keywords:
```
{'False',
 'None',
 'True',
 'and',
 'assert',
 'break',
 'continue',
 'del',
 'finally',
 'global',
 'lambda',
 'nonlocal',
 'or',
 'pass',
 'try',
 'with'}
```

**Comment by cdgriffith** [comment]
Thank you so much! I had no idea this existed (I tried searching for similar but 'python built-ins' and similar searches, but were way to broad.)

If you could please, make your PR against `development` (and add your name to the `code contribution` section of the AUTHORS file) and I will merge.

**Comment by dhilipsiva** [comment]
Closing this in the favour of #30

**Comment by dhilipsiva** [comment]
@cdgriffith Even I did not knew that. I just asked google for "[show reserved keywords python](https://www.google.co.in/search?q=show+reserved+keywords+python)" and there it was 😝

**Pull request closed without merging**